docker/server/entrypoint.sh

```diff
@@ -30,6 +30,36 @@ chown -R $USER:$GROUP \
     "$TMP_DIR" \
     "$USER_PATH"
 
+if [ -n "$(ls /docker-entrypoint-initdb.d/)" ]; then
+    gosu clickhouse /usr/bin/clickhouse-server --config-file=$CLICKHOUSE_CONFIG &
+    pid="$!"
+    sleep 1
+
+    clickhouseclient=( clickhouse client --multiquery )
```

Comment:
Don't understand, why we need array here.

Reply:
I see. In the [postgres/mysql entrypoints](https://github.com/docker-library/postgres/blob/master/docker-entrypoint.sh#L129) password and database added later in script. This can be useful for future `*_PASSWORD` and `*_USER` [variables](https://hub.docker.com/_/postgres/) support.

Array is redundant right now. I can change it to simple variable.